### PR TITLE
feat: Exposes store to be accessible through the wasm

### PR DESCRIPTION
**Pull request by soluchok**
Signed-off-by: Andrii Soluk <isoluchok@gmail.com>

### CODE REVIEW

**Review by rolsonquadras — APPROVED** [review]

File: pkg/controller/command/store/command.go [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+		return command.NewValidationError(InvalidRequestErrorCode, err)
+	}
+
+	records := c.store.Iterator(request.StartKey, request.EndKey)
+	defer records.Release()
+
+	var values [][]byte
+
+	for records.Next() {
+		values = append(values, records.Value())
```
Comment (llorllale) [inline pkg/controller/command/store/command.go:150]: this will load all records in memory.

**Review by fqutishat — APPROVED** [review]

**Review by DRK3 — APPROVED** [review]

**Review by llorllale — APPROVED** [review]
as per @DRK3 

> temporary solution to allow us to migrate any remaining edge-agent direct-to-IndexedDB into aries storage (EDV) by just updating the KeyValueStore class in edge-agent

File: pkg/controller/command/store/command.go (hunk 1 above)
Comment (soluchok) [reply pkg/controller/command/store/command.go:150]: I know :( But I need this at least for this release.

**Comment by DRK3** [comment]
Clarification to what I said earlier (in the quote that @llorllale posted): This general storage API may stay, but what we should change eventually is doing individual refactoring of each thing that is using this general storage. Some of the things (e.g. `DIDMetadata`) are actually mostly redundant with what's already being stored via the Aries SaveDID method.

**Pull request merged**